src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,66 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotations
+        type: array
+        items:
+            type: object
+            properties:
+                id:
+                    description: Annotation ID
+                    type: string
+                access:
+                    $ref: ../misc/access-level.v1.yaml
+                parents:
+                    description: Array of parent annotation ID's
+                    type: array
+                    items:
+                        type: string
+                document:
```

Comment:
Not a fan of these, would much rather this API translated it into our concepts (in this case content item). But it does allow us to do what we need (and redirects should cover any changes), so I can live with it (for now).